src/content/docs/apm/apm-ui-pages/monitoring/find-performance-problems-apm-transactions-page.mdx

```diff
@@ -0,0 +1,374 @@
+---
+title: 'Find performance problems using the APM Transactions page'
+tags:
+  - APM
+  - APM UI pages
+  - Monitoring
+translate:
+  - jp
+metaDescription: APM's Transactions page shows app, browser, and other transaction details, average response time, and throughput.
+redirects:
+  - /docs/applications-menu/web-transactions
+  - /docs/applications-menu/background-tasks
+  - /docs/applications-menu/transactions-dashboard
+  - /docs/apm/applications-menu/monitoring/transactions-dashboard
+  - /docs/apm/applications-menu/monitoring/transactions-page
+  - /docs/apm/applications-menu/monitoring/transactions-page-find-performance-problems
+  - /docs/apm/applications-menu/monitoring/transactions-page-find-specific-performance-problems
+  - /docs/apm/transactions/intro-transactions/transactions-page-find-specific-performance-problems
+  - /docs/apm/apm-ui-pages/monitoring/transactions-page-find-specific-performance-problems
+---
+
+The APM **Transactions** page helps you identify transactions to look at more closely in order to troubleshoot performance problems and avoid errors. 
+
+In general, the longer a given transaction takes, and the more time it takes compared to other transactions, the more likely it is that it's a source of issues for a given app. Similarly, low throughput can be an indicator that not everything is working as it should. 
```

Comment:
I'd expand on this a bit, since "issues" could mean a lot. Perhaps something like "Transactions that take more time than expected to complete are more likely to be a source of frustration for end users, or contribute to a failure to meet service level objectives (SLOs). Lower-than-expected throughput could also indicate a problem with traffic reaching a service, or insufficient resources to handle incoming requests."